scriptmodules/emulators/mupen64plus.sh

```diff
@@ -370,6 +370,13 @@ function configure_mupen64plus() {
             setAutoConf mupen64plus_audio 1
             setAutoConf mupen64plus_compatibility_check 1
         elif isPlatform "mesa"; then
+            # Create Video-Rice section in .cfg
+            if ! grep -q "\[Video-Rice\]" "$config"; then
+                echo "[Video-Rice]" >> "$config"
+            fi
+            # Fix #3138
```

Comment:
Please can you add a short description instead. The commit can reference the ticket.